packages/api-security/src/plugins/models/securityUser.model.ts

```diff
@@ -2,22 +2,24 @@ import { flow } from "lodash";
 import { validation } from "@webiny/validation";
 import md5 from "md5";
 import bcrypt from "bcryptjs";
-import { withHooks, withProps, withName, string, withFields, onSet, ref } from "@webiny/commodo";
+import {
+    withHooks,
+    withProps,
+    withName,
+    string,
+    withFields,
+    onSet,
+    ref,
+    skipOnPopulate,
+} from "@webiny/commodo";
 
-export default ({
-    createBase,
-    SecurityRole,
-    SecurityRoles2Models,
-    SecurityGroup,
-    SecurityGroups2Models,
-    context
-}): any => {
+export default ({ createBase, context }): any => {
     // TODO: figure out how to create typings for a `model`
     const SecurityUser: any = flow(
```

Comment:
Use `pipe` (imported from @webiny/commodo) instead of `flow` (it's a new addition).